vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/index/updateIndex/IndexUpdater.java

```diff
@@ -86,12 +86,6 @@ public  IndexUpdater(String indexFolder, IPointable result, UTF8StringPointable
         this.pathsFromFileList = new HashSet<>();
     }
 
-    /**
-     * Perform the initial configuration for index update/ delete processes.
```

Comment:
I think we should keep these comments. They seem to add to the documentation.
Same goes for the next three deleted comments.